runtime/gc_glue_java/HeapWalkerDelegate.cpp

```diff
@@ -60,14 +60,16 @@ MM_HeapWalkerDelegate::doContinuationNativeSlots(MM_EnvironmentBase *env, omrobj
 {
 	J9VMThread *currentThread = (J9VMThread *)env->getLanguageVMThread();
 
-	if (MM_GCExtensions::needScanStacksForContinuationObject(currentThread, objectPtr)) {
+	const bool isGlobalGC = false;
```

Comment:
This is weird one, since Heap walker is not a GC, to start with.
Either false or true is probably good, but since this it's actually (so far) only instantiated by ParallelGlobalGC and used for whole heap walk, it'd actually mark it as global.
If in future it's used from different GCs, then we would have to resolve this in run-time - similar how we do it for WriteOnceCompactor.

Reply:
seems like you forgot to flip this to true

Reply:
this has not been fixed yet